### PR TITLE
converts unnecessary capitals in names to lowercase

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

These things annoy me. Names shouldn't be capitalised unless they're... well, names, or otherwise have a reason to be that way.
I searched with `^/obj/item.*\n	name = ".*[A-Z][a-z]` for this.

I probably missed some bits. They can be caught as and when.

## Changelog
:cl:
spellcheck: Fixed a few incorrect capitalisations
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm [hunk 1]
```diff
@@ -52,12 +52,12 @@
...
 /obj/item/ammo_box/c32/rubber
-	name = "ammo box (.32 Rubber)"
+	name = "ammo box (.32 rubber)"
 	ammo_type = /obj/item/ammo_casing/c32/rubber
 
 /obj/item/ammo_box/c32/fire
-	name = "ammo box (.32 Incen)"
-	ammo_type = /obj/item/ammo_casing/c32_incendiary
+	name = "ammo box (.32 incendiary)"
+	ammo_type = /obj/item/ammo_casing/c32_incendiarydiary
```
Comment (mcmeiler) [inline modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm]: Looks like the regex messed up here with editing a typepath.

File: modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm [hunk 2]
```diff
@@ -71,12 +71,12 @@
...
 /obj/item/ammo_box/c34/rubber
-	name = "ammo box (.34 Rubber)"
+	name = "ammo box (.34 rubber)"
 	ammo_type = /obj/item/ammo_casing/c34/rubber
 
 /obj/item/ammo_box/c34/fire
-	name = "ammo box (.34 Incen)"
-	ammo_type = /obj/item/ammo_casing/c34_incendiary
+	name = "ammo box (.34 incendiary)"
+	ammo_type = /obj/item/ammo_casing/c34_incendiarydiary
```
Comment (mcmeiler) [inline modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm]: ditto

**Review by mcmeiler — COMMENTED** [review]
I just looked at the CI, you probably know the drill anyway.

File: modular_skyrat/modules/cellguns/code/mediguns.dm [hunk 3]
```diff
@@ -274,13 +274,13 @@
 	ammo_type = /obj/item/ammo_casing/energy/medical/utility/body_teleporter
 
 /obj/item/weaponcell/medical/utility/relocation
-	name = "Oppressive Force Relocation Medicell"
+	name = "oppressive force relocation medicell"
```
Comment (theselfish) [inline modular_skyrat/modules/cellguns/code/mediguns.dm]:
```suggestion
	name = "Oppressive Force relocation medicell"
```
It's entirely up to you, but I think this looks better.

File: modular_skyrat/modules/cellguns/code/mediguns.dm [hunk 4]
```diff
@@ -274,13 +274,13 @@
...
 	desc = "A medicell that safely relocates personnel"
 	icon_state =  "body"
 	ammo_type = /obj/item/ammo_casing/energy/medical/utility/relocation/standard
 
 /obj/item/weaponcell/medical/utility/relocation/upgraded
-	name = "Upgraded Oppressive Force Relocation Medicell"
+	name = "upgraded oppressive force relocation medicell"
```
Comment (theselfish) [inline modular_skyrat/modules/cellguns/code/mediguns.dm]:
```suggestion
	name = "upgraded Oppressive Force relocation medicell"
```
Ditto.

File: modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm [hunk 5]
```diff
@@ -115,16 +115,16 @@
...
 
 /obj/item/ammo_box/b10mm
-	name = "ammo box (10mm Auto)"
+	name = "ammo box (10mm auto)"
 	ammo_type = /obj/item/ammo_casing/b10mm
 	max_ammo = 20
 
 /obj/item/ammo_box/b10mm/hp
-	name = "ammo box (10mm Auto HP)"
+	name = "ammo box (10mm auto HP)"
```
Comment (theselfish) [inline modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm]: HP should be changed to whatever it's long form is.

File: modular_skyrat/modules/sec_haul/code/guns/bullets.dm [hunk 6]
```diff
@@ -130,37 +130,37 @@
...
-	name = "10mm Auto bullet"
+	name = "10mm auto bullet"
 	damage = 27
 	speed = 1
 	wound_bonus = 5
 
 /obj/item/ammo_casing/b10mm/hp
-	name = "10mm Auto JHP casing"
-	desc = "A 10mm Auto JHP bullet casing."
+	name = "10mm auto JHP casing"
```
Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/guns/bullets.dm:147]: No one but gun nerds actually know what a JHP is.

File: modular_skyrat/modules/sec_haul/code/guns/bullets.dm [hunk 7]
```diff
@@ -130,37 +130,37 @@
...
+	desc = "A 10mm auto JHP bullet casing."
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
 	icon_state = "sh-casing"
 	caliber = CALIBER_10MMAUTO
 	projectile_type = /obj/projectile/bullet/advanced/b10mm/hp
 
 /obj/projectile/bullet/advanced/b10mm/hp
-	name = "10mm Auto JHP bullet"
+	name = "10mm auto JHP bullet"
```
Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/guns/bullets.dm:155]: No one but gun nerds actually know what a JHP is.

File: modular_skyrat/modules/sec_haul/code/guns/bullets.dm [hunk 8]
```diff
@@ -182,7 +182,7 @@
 	embedding = null
 
 /obj/item/ammo_casing/b10mm/ihdf
-	name = "10mm Auto IHDF bullet casing"
+	name = "10mm auto IHDF bullet casing"
```
Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/guns/bullets.dm:185]: Ditto.

File: modular_skyrat/modules/sec_haul/code/guns/bullets.dm [hunk 9]
```diff
@@ -191,7 +191,7 @@
 	harmful = FALSE
 
 /obj/projectile/bullet/advanced/b10mm/ihdf
-	name = "10mm Auto ihdf bullet"
+	name = "10mm auto ihdf bullet"
```
Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/guns/bullets.dm:194]: Ditto.

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 10]
```diff
@@ -506,7 +506,7 @@
 	company_flag = COMPANY_ARMADYNE
 
 /obj/item/ammo_box/magazine/dmr
-	name = ".557 DMR Magazine"
+	name = ".557 DMR magazine"
```
Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm]: Ditto.

File: modular_skyrat/modules/sec_haul/code/guns/guns.dm [hunk 11]
```diff
@@ -859,7 +859,7 @@
 	AddComponent(/datum/component/scope, range_modifier = 1.75)
 
 /obj/item/ammo_box/magazine/multi_sprite/norwind
-	name = "12.7x30mm DMR Magazine"
+	name = "12.7x30mm DMR magazine"
```
Comment (theselfish) [inline modular_skyrat/modules/sec_haul/code/guns/guns.dm:863]: Ditto.

File: modular_skyrat/modules/cellguns/code/mediguns.dm (hunk 3 above)
Comment (tf-4) [reply modular_skyrat/modules/cellguns/code/mediguns.dm]: agree actually.

File: modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm (hunk 1 above)
Comment (tf-4) [inline modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm]:
```suggestion
	ammo_type = /obj/item/ammo_casing/c32_incendiary
```

File: modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm (hunk 2 above)
Comment (tf-4) [inline modular_skyrat/modules/gun_cargo/code/objects/ammo_boxes.dm]:
```suggestion
	ammo_type = /obj/item/ammo_casing/c34_incendiary
```

**Comment by tf-4** [comment]
While I agree those gun acronyms should be changed, I don't even know what half of them mean and it's not really within the purpose of this PR. I might look to do that in a new PR.

**Comment by theselfish** [comment]
Very fair.

I like Oppressive Force capitalised as kind of a 'brand name' if that makes sense.

**Comment by Tastyfish** [comment]
I'd argue that "Freeman's Crowbar" in its entirety is a proper name of a unique artifact.

**Pull request merged**